scalatest/src/main/scala/org/scalatest/tools/Framework.scala

```diff
@@ -791,16 +795,17 @@ class Framework extends SbtFramework {
         throw new IllegalStateException("done method is called twice")
     }
 
-    def args = runArgs
+    def args = runArgs.asInstanceOf[Array[String | Null]]
```

Comment:
**Overridden java array field**
A Scala field overrides some Java array field. We need to use `Array[T | Null]` instead of `Array[T]` for the overriding to work.